app/scripts/controllers/journalUpload.js

```diff
@@ -76,8 +77,10 @@ angular
           //scroll to top on page change
           $window.scrollTo(0, 0);
         }
-
-        $scope.refresh();
+        // Only refresh when paramters are different
+        if (newParameters !== oldParameters) {
+          $scope.refresh();
```

Comment:
This was always just refreshing no matter what, so on page load, there was an additional API call for account transfers.